docs/DOCUMENTATION.md

```diff
@@ -524,6 +524,9 @@ const graph = {
         This is how link strokeWidth will be calculated:`javascript strokeWidth += (linkValue * strokeWidth) / 10;` (optional, default `false`)
     -   `link.strokeWidth` **[number][181]** <a id="link-stroke-width" href="#link-stroke-width">🔗</a> 🔍🔍🔍 strokeWidth for all links. By default the actual value is obtain by the
         following expression:`javascript link.strokeWidth * (1 / transform); // transform is a zoom delta Δ value` (optional, default `1.5`)
+    -   `link.strokeDasharray` **[string][183]** <a id="link-stroke-dasharray" href="#link-stroke-dasharray">🔗</a> 🔍🔍🔍 strokeDasharray for all links.
```

Comment:
Please drop these changes. Documentation is only updated upon releasing a new version. You can reuse these lines of documentation to add it on [graph.config.js](https://github.com/danielcaldas/react-d3-graph/blob/eb1c81dfeb3835bb87d10dd570ecbec236f53157/src/components/graph/graph.config.js#L240)

Reply:
Also please make sure to wrap each one of the properties (e.g. `strokeDasharray`) with a link to the respective MDN page. example:

```
<a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray">strokeDasharray</a>
```

This is extremely valuable for navigability and to make sure other developers don't mix this with something else.